dist/boards/slstk3400a/doc.txt

```diff
@@ -22,20 +22,20 @@ actively measure the power consumption of your hardware and code, in real-time.
 | RAM             | 8.0KiB                                               |
 | Flash           | 64.0KiB                                              |
 | EEPROM          | no                                                   |
-| Frequency       | up to 24 MHz                                         |
+| Frequency       | up to 25 MHz                                         |
 | FPU             | no                                                   |
 | MPU             | no                                                   |
-| DMA             | 8 channels                                           |
-| Timers          | 3x 16-bit + 1x 16-bit (low power)                    |
+| DMA             | 6 channels                                           |
+| Timers          | 2x 16-bit + 1x 16-bit (low power)                    |
 | ADCs            | 12-bit ADC                                           |
-| UARTs           | 2x USART, 3x UART, 1x LEUART                         |
+| UARTs           | 2x USART, 1x LEUART                                  |
 | SPIs            | 2x USART                                             |
 | I2Cs            | 1x                                                   |
 | Vcc             | 1.98V - 3.8V                                         |
-| Datasheet       | [Datasheet](https://www.silabs.com/documents/public/data-sheets/efm32hg-datasheet.pdf)                                   |
-| Manual          | [Manual](https://www.silabs.com/documents/public/reference-manuals/efm32hg-rm.pdf)                                       |
-| Board Manual    | [Board Manual](https://www.silabs.com/documents/public/user-guides/ug255-stk3400-user-guide.pdf)                         |
-| Board Schematic | Can be downloaded using Silicon Labs' Simplicity Studio         |
+| Datasheet       | [Datasheet](https://www.silabs.com/documents/public/data-sheets/efm32hg-datasheet.pdf)                    |
+| Manual          | [Manual](https://www.silabs.com/documents/public/reference-manuals/efm32hg-rm.pdf)                        |
+| Board Manual    | [Board Manual](https://www.silabs.com/documents/public/user-guides/ug255-stk3400-user-guide.pdf)          |
+| Board Schematic | [Board Schematic](https://www.silabs.com/documents/public/schematic-files/efm32hg-stk3400-schematics.pdf) |
```

Comment:
I intentionally removeded links to the schematics, because they are not available for all boards. Also, I noted that the links break frequently, so I won't provde schematics anymore.

Reply:
Updated.